fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -1149,6 +1201,7 @@ public static class Builder {
         private boolean useSynchronizedSession = true;
         private long leaseLengthMillis = DEFAULT_LEASE_LENGTH_MILLIS;
         private long timeLimitMilliseconds = 0;
+        private long transactionTimeQuotaMilliseconds = 0;
```

Comment:
Should this pick a number like 4,000 ms as its default? On the one hand, leaving it as unlimited is consistent with the current behavior; on the other hand, due to the FDB 5-second transaction limit, any value above, like, 4.9 seconds is not usually all that useful

Reply:
I like this idea of setting a very practical default! 
✅